recipes/ez-zarr/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ez-zarr" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/ez_zarr-{{ version }}.tar.gz
+  sha256: 2258663f4d7fa929ada9d383e6f4bf9bbf205ef55776c7ca74e4d18db5054919
+
+build:
+  entry_points:
+    - ez_zarr = ez_zarr.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools
+    - wheel
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.9
+    - anndata
+    - dask-core
+    - numpy
+    - pandas
+    - zarr
+    - matplotlib-base
+    - scikit-image
+
+test:
+  imports:
+    - ez_zarr
+  commands:
+    - pip check
+    - ez_zarr --help
+  requires:
+    - pip
+    - pytest
```

Comment:
This is not used in the tests
```suggestion
```

Reply:
Yes indeed. I have removed it. Thank you @xhochy